src/telemetry/metadata.h

```diff
@@ -1,17 +0,0 @@
-/*
```

Comment:
Why isn't this file simply renamed instead of deleted and re-added? Now there's an unnecessarily large diff.

You should try to do 

```
git mv  src/telemetry/metadata.h src/telemetry/telemetry_metadata.h
```

Same goes for `metadata.c`.

(Actually unclear why these needs to be renamed since already namespaced by the telemetry dir).

Reply:
This actually was done as a `git mv`, and was shown as a rename in my local `git status`.  Somehow that got lost when I pushed the change to github.

Reply:
can it be fixed somehow. I think it helps things like rebasing other branches.